src/lib/db/interface.ts

```diff
@@ -20,9 +20,10 @@ export interface User {
 
 const createTodoTable = `create table if not exists todos (
 	id integer not null primary key,
-  title text unique not null,
+  title text not null,
 	content text not null,
 	created_at text default current_timestamp,
+  status integer not null,
```

Comment:
Your idea is right
Hehe, we can't do like this, undo all changes in this file.
I will tell you how to change later.